src/algebra/module/equiv.lean

```diff
@@ -539,6 +539,18 @@ end add_comm_monoid
 
 end linear_equiv
 
+namespace linear_equiv
+variables {𝕜 α β : Type*} [semiring 𝕜] [add_comm_group α] [add_comm_group β] [module 𝕜 α]
+  [module 𝕜 β]
+
+@[simp]
+lemma subgroup_map_symm_apply (e : α ≃ₗ[𝕜] β) {L : add_subgroup α} {g : L.map (e : α →+ β)} :
```

Comment:
I'm very confused by this lemma name; shouldn't it be about `linear_equiv.submodule_map`?

Reply:
No, the point is to use `linear_equiv.symm`. We can't use `add_equiv.add_subgroup_symm_apply` here.

There's a typo, though.